### PR TITLE
BI-7907 Define AWS user responsible for uploading results

### PR DESCRIPTION
**Pull request by dlgroves-ch**
* Define a new AWS user used by the app to upload results to S3.

### CODE REVIEW

File: groups/data-reconciliation/module-iam/main.tf [hunk 1]
```diff
@@ -83,3 +83,51 @@ data "aws_iam_policy_document" "cloudwatch-policy" {
   }
 }
 
+# IAM user responsible for uploading results to S3
+resource "aws_iam_user" "iam-user" {
```
Comment (Jata2k) [inline groups/data-reconciliation/module-iam/main.tf:87]: Could this be done through the task role instead of defining a user to do so?

Comment (dlgroves-ch) [reply groups/data-reconciliation/module-iam/main.tf:87]: The application requires an access key id and a secret access key in order to work. I'd be happy to put a task in the backlog to rework this later on, but we are working to tight deadlines.

**Review by kinpang-CH — APPROVED** [review]

File: groups/data-reconciliation/module-iam/main.tf (hunk 1 above)
Comment (dlgroves-ch) [reply groups/data-reconciliation/module-iam/main.tf:87]: I've raised BI-8463 to rework the application to use AWS_CONTAINER_CREDENTIALS_RELATIVE_URI.

Comment (Jata2k) [reply groups/data-reconciliation/module-iam/main.tf:87]: If we put a story in the backlog to revisit this that would be fine

**Review by Jata2k — APPROVED** [review]

**Pull request merged**